### DISCUSSION TITLE
With draft time looming, is there a way to sync this api with our drafts?

### DISCUSSION THREAD

**Discussion by Penetics** [cwendt94/espn-api#551] [Q&A]
I.e. can i get the specific league rules (i.e. PPR vs 0.5 PPR), player stats, and which players are available?

**Comment by cwendt94** [comment]
You can get league rules and player stats however you won't be bale to see which players are available at the time of the draft. During the draft ESPN doesn't provide any API's to get details/players. However at the draft is finished you can get all that information including free agents.

**Reply by ChandlerCGray** [reply]
Are you saying that the league.teams does not get assigned to players _during_ the draft? We have to wait until the draft is over for that to be updated? Specifically I'm asking for the Baseball part.

**Reply by cwendt94** [reply]
That is correct

**Reply by hydephilip01** [reply]
> That is correct

Are there any plans to implement real time updates to teams/draft picks during the draft?

**Comment by shaggycamel** [comment]
@cwendt94 

Aware of this chrome extension?
 https://chromewebstore.google.com/detail/ljnelhiofbippkmbeioamhnjihkecgbj?utm_source=item-share-cp

Reckon you can pick it apart? Or, ESPN got live draft data on lockdown

**Reply by cwendt94** [reply]
I was not aware of this but would definitely be interested to see how they parsed ESPN's live draft.
